normandy/control/templates/control/create_recipe.html

```diff
@@ -1,18 +1,128 @@
-  </div>
+{% block page-header %}
+  <h2>Create Recipe</h2>
 {% endblock %}
 
 {% block content %}
-  <div class="fluid-8">
-    <h2>new recipe</h2>
-    <form action="" method="post">
+  <div class="fluid-7">
+    <form id="add-recipe-form" class="crud-form" action="" method="post">
       {% csrf_token %}
-      {{ form.as_p }}
-      <input type="submit" value="Save">
+      {{ form.non_field_errors }}
+
+      <div class="row">
+        <div class="fluid-3">
+          {{ form.name.errors }}
+          <label for="{{ form.name.id_for_label }}">Name:</label>
+          {{ form.name }}
+        </div>
+
+        <div class="fluid-3 float-right">
+          {{ form.enabled.errors }}
+          {{ form.enabled }}
+          <div class="checkbox-container {{ recipe.enabled|yesno:'green,red' }}">
+            <span {{ recipe.enabled|yesno:',hidden' }}><i class="fa fa-check fa-lg pre"></i> Enabled</span>
+            <span {{ recipe.enabled|yesno:'hidden,' }}><i class="fa fa-times fa-lg pre"></i> Disabled</span>
+          </div>
+        </div>
+      </div>
+
+      <div class="row">
+        <div class="fluid-3">
+          {{ form.action.errors }}
+          <label for="{{ form.action.id_for_label }}">Action:</label>
+          {{ form.action }}
+
+          <!-- hiding arguments_json until the action form rendering is finalized -->
+          <div hidden>
+            {{ form.arguments_json }}
+          </div>
+        </div>
+      </div>
+
+      <div class="row">
+        <div class="fluid-3">
+          {{ form.sample_rate.errors }}
+          <label for="{{ form.sample_rate.id_for_label }}">Sample Rate:</label>
+          {{ form.sample_rate }}
+          <p class="help">{{ form.sample_rate.help_text }}</p>
+        </div>
+      </div>
+
+      <div class="row">
+        <div class="fluid-3">
+          {{ form.release_channels.errors }}
+          <label for="{{ form.release_channels.id_for_label }}">Release Channels:</label>
+          {{ form.release_channels }}
+        </div>
+      </div>
+
+      <div class="row">
+        <div class="fluid-3">
+          {{ form.start_time.errors }}
+          <label for="{{ form.start_time.id_for_label }}">Start Time:</label>
+          <div class="input-with-icon calendar">
+            {{ form.start_time }}
+          </div>
+        </div>
+
+        <div class="fluid-3 float-right">
+          {{ form.end_time.errors }}
+          <label for="{{ form.end_time.id_for_label }}">End Time:</label>
+          <div class="input-with-icon calendar">
+            {{ form.end_time }}
+          </div>
+        </div>
+      </div>
+
+      <div class="row">
+        <div class="fluid-3">
+          {{ form.locales.errors }}
+          <label for="{{ form.locales.id_for_label }}">Available Locales:</label>
+          {{ form.locales }}
+          <a href="#" class="batch-select">Select all <i class="fa fa-angle-right post"></i></a>
```

Comment:
Yea, these are still static. I'm fussing with a couple of different react components, but getting all of those in and working would make this PR a bit larger than I'd like. Same with date/time picker for start & end time fields.